src/traces/scattersmith/attributes.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var hovertemplateAttrs = require('../../plots/template_attributes').hovertemplateAttrs;
+var texttemplateAttrs = require('../../plots/template_attributes').texttemplateAttrs;
+var extendFlat = require('../../lib/extend').extendFlat;
+var scatterAttrs = require('../scatter/attributes');
+var lineAttrs = scatterAttrs.line;
+
+module.exports = {
+    mode: scatterAttrs.mode,
+
+    re: {
+        valType: 'data_array',
+        editType: 'calc+clearAxisTypes',
+        description: 'Sets the real coordinates'
+    },
+
+    im: {
+        valType: 'data_array',
+        editType: 'calc+clearAxisTypes',
+        description: 'Sets the imaginary coordinates'
+    },
+
+    text: scatterAttrs.text,
+    texttemplate: texttemplateAttrs({editType: 'plot'}, {
+        keys: ['re', 'im', 'text']
+    }),
+    hovertext: scatterAttrs.hovertext,
+
+    line: {
+        color: lineAttrs.color,
+        width: lineAttrs.width,
+        dash: lineAttrs.dash,
+        shape: extendFlat({}, lineAttrs.shape, {
+            values: ['linear', 'spline']
+        }),
+        smoothing: lineAttrs.smoothing,
+        editType: 'calc'
+    },
+    connectgaps: scatterAttrs.connectgaps,
+
+    marker: scatterAttrs.marker,
+    cliponaxis: extendFlat({}, scatterAttrs.cliponaxis, {dflt: false}),
+
+    textposition: scatterAttrs.textposition,
+    textfont: scatterAttrs.textfont,
+
+    fill: extendFlat({}, scatterAttrs.fill, {
+        values: ['none', 'toself', 'tonext'],
+        dflt: 'none',
+        description: [
+            'Sets the area to fill with a solid color.',
+            'Use with `fillcolor` if not *none*.',
+            'scattersmith has a subset of the options available to scatter.',
+            '*toself* connects the endpoints of the trace (or each segment',
+            'of the trace if it has gaps) into a closed shape.',
+            '*tonext* fills the space between two traces if one completely',
+            'encloses the other (eg consecutive contour lines), and behaves like',
+            '*toself* if there is no trace before it. *tonext* should not be',
+            'used if one trace does not enclose the other.'
+        ].join(' ')
+    }),
+    fillcolor: scatterAttrs.fillcolor,
+
+    hoveron: scatterAttrs.hoveron,
+    hovertemplate: hovertemplateAttrs(),
+
+    selected: scatterAttrs.selected,
```

Comment:
Selection used to be working. But now it is not working. Could you please bring it back?

Reply:
You need to revert https://github.com/plotly/plotly.js/pull/5615/commits/71eedaabe5deb2da7aae13ae8f2ec96e7b2d62de.

Reply:
012845210a7f3416c9b8c62efc0ad98a3bae30c9